client/analytics/report/get-reports.js

```diff
@@ -107,7 +107,7 @@ export default () => {
 			report: 'customers',
 			title: __( 'Customers', 'woocommerce-admin' ),
 			component: CustomersReport,
-			id: 'woocommerce-analytics-customers',
+			id: null,
```

Comment:
Weird, so this `id` is purely for slotFill. I'm confused why removing it would prevent a double nav item.

Reply:
Since WCA registers 2 "Customers" menu items, both with the same ID, they fill the same slot.  By default a slot can be filled any number of times.

It may be worth considering limiting this to a single fill, but in this case I think we should favor the menu item with `/customers` instead of the one with a path of `/analytics/customers` due to its location in the new nav.